src/platform/api.py

```diff
@@ -108,3 +111,41 @@ def get_variables_and_secrets() -> list[dict]:
     response = _make_get_request(f"/computebenches/{_get_org_id()}/placeholder/secrets")
 
     return response["data"]
+
+
+@functools.cache
+@beartype
+def get_public_keys() -> list[dict]:
+    """get public keys from public endpoint"""
+
+    jwks_url = urljoin(get_value()["auth_domain"], ".well-known/jwks.json")
```

Comment:
IMO, you should push `get_value()["auth_domain"]` as far up the call stack as possible. Not blocking.

Reply:
i get you now. `get_value` is expected to work in tests too -- we set up a dummy config file